### ISSUE TITLE
3.0.2 break meteor

### ISSUE THREAD

**Issue by chongwang87** [okgrow/analytics#199]
after update to 3.0.2, meteor failed to startup. showing error as below

downgrade to 3.0.1 work fine

````
/Users/keep/.meteor/packages/tap_i18n/.1.8.2.14jbfa.tp1jg++os+web.browser+web.cordova/plugin.tap-i18n-compiler.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:116
      throw error;
      ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:28:11)
    at Object.relative (path.js:1269:5)
    at /Users/keep/.meteor/packages/meteor-tool/.1.6.0-rc.6.av6hsk++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/tools/static-assets/server/mini-files.js:109:14
    at ImportScanner._getRelativeImportId (/tools/isobuild/import-scanner.js:599:43)
    at each (/tools/isobuild/import-scanner.js:583:33)
    at _.each._.forEach (/Users/keep/.meteor/packages/meteor-tool/.1.6.0-rc.6.av6hsk++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:158:9)
    at ImportScanner._resolve (/tools/isobuild/import-scanner.js:572:7)
    at each (/tools/isobuild/import-scanner.js:665:29)
    at _.each._.forEach (/Users/keep/.meteor/packages/meteor-tool/.1.6.0-rc.6.av6hsk++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:158:9)
    at ImportScanner._scanFile (/tools/isobuild/import-scanner.js:656:5)
    at each (/tools/isobuild/import-scanner.js:746:12)

````

**Comment by jancurn** [comment]
Same here, we're getting the following error:

```
/Users/jan/.meteor/packages/less/.2.7.9.uqarj0++os+web.browser+web.cordova/plugin.compileLessBatch.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:190
      throw error;
      ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:8:11)
    at Object.posix.relative (path.js:496:3)
    at /Users/jan/.meteor/packages/meteor-tool/.1.5.2_2.xs0smq++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/tools/static-assets/server/mini-files.js:109:14
    at ImportScanner._getRelativeImportId (/tools/isobuild/import-scanner.js:525:43)
    at /tools/isobuild/import-scanner.js:509:33
    at _.each._.forEach (/Users/jan/.meteor/packages/meteor-tool/.1.5.2_2.xs0smq++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:87:22)
    at ImportScanner._resolve (/tools/isobuild/import-scanner.js:498:7)
    at /tools/isobuild/import-scanner.js:564:29
    at _.each._.forEach (/Users/jan/.meteor/packages/meteor-tool/.1.5.2_2.xs0smq++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:87:22)
    at ImportScanner._scanFile (/tools/isobuild/import-scanner.js:557:5)
    at /tools/isobuild/import-scanner.js:670:12
    at _.each._.forEach (/Users/jan/.meteor/packages/meteor-tool/.1.5.2_2.xs0smq++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:87:22)
    at ImportScanner._scanFile (/tools/isobuild/import-scanner.js:557:5)
    at /tools/isobuild/import-scanner.js:670:12
    at _.each._.forEach (/Users/jan/.meteor/packages/meteor-tool/.1.5.2_2.xs0smq++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:87:22)
    at ImportScanner._scanFile (/tools/isobuild/import-scanner.js:557:5)
    at /tools/isobuild/import-scanner.js:670:12
    at _.each._.forEach (/Users/jan/.meteor/packages/meteor-tool/.1.5.2_2.xs0smq++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:87:22)
    at ImportScanner._scanFile (/tools/isobuild/import-scanner.js:557:5)
    at /tools/isobuild/import-scanner.js:670:12
    at _.each._.forEach (/Users/jan/.meteor/packages/meteor-tool/.1.5.2_2.xs0smq++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:87:22)
    at ImportScanner._scanFile (/tools/isobuild/import-scanner.js:557:5)
    at /tools/isobuild/import-scanner.js:357:14
    at Array.forEach (native)
    at ImportScanner.scanImports (/tools/isobuild/import-scanner.js:355:22)
    at /tools/isobuild/compiler-plugin.js:1040:17
    at Array.forEach (native)
    at Function.computeJsOutputFilesMap (/tools/isobuild/compiler-plugin.js:1008:19)
    at ClientTarget._emitResources (/tools/isobuild/bundler.js:1059:8)
    at /tools/isobuild/bundler.js:830:12
    at /tools/utils/buildmessage.js:359:18
    at [object Object]._.extend.withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:352:34
    at [object Object]._.extend.withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:350:23
    at [object Object]._.extend.withValue (/tools/utils/fiber-helpers.js:89:14)
    at Object.enterJob (/tools/utils/buildmessage.js:324:26)
    at ClientTarget.make (/tools/isobuild/bundler.js:821:18)
    at /tools/isobuild/bundler.js:2918:14
    at /tools/isobuild/bundler.js:3007:20
    at Array.forEach (native)
    at Function._.each._.forEach (/Users/jan/.meteor/packages/meteor-tool/.1.5.2_2.xs0smq++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/underscore/underscore.js:79:11)
    at /tools/isobuild/bundler.js:3006:7
    at /tools/utils/buildmessage.js:271:13
    at [object Object]._.extend.withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:264:29
    at [object Object]._.extend.withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:262:18
    at [object Object]._.extend.withValue (/tools/utils/fiber-helpers.js:89:14)
    at /tools/utils/buildmessage.js:253:23
    at [object Object]._.extend.withValue (/tools/utils/fiber-helpers.js:89:14)
    at Object.capture (/tools/utils/buildmessage.js:252:19)
    at bundle (/tools/isobuild/bundler.js:2899:31)
    at /tools/isobuild/bundler.js:2846:32
    at Object.withCache (/tools/fs/files.js:1694:12)
    at Object.exports.bundle (/tools/isobuild/bundler.js:2846:16)
    at /tools/runners/run-app.js:583:36
    at Function.run (/tools/tool-env/profile.js:490:12)
    at bundleApp (/tools/runners/run-app.js:582:34)
    at [object Object]._.extend._runOnce (/tools/runners/run-app.js:626:35)
    at [object Object]._.extend._fiber (/tools/runners/run-app.js:885:28)
    at /tools/runners/run-app.js:412:12
```

We have the latest version of everything.

**Comment by cfnelson** [comment]
@chongwang87 @jancurn  Please revert to `v3.0.1`. Closing this issue for the original issue opened, we are tracking the problem here at https://github.com/okgrow/analytics/issues/197 and opened an issue over on meteor https://github.com/meteor/meteor/issues/9193

**Comment by cfnelson** [comment]
@chongwang87 @jancurn We have released `v3.0.3` please update to the latest and everything should be working again! 🎉  😃  The update also includes two fixes to the `data.title` for usage with react-router v4 and the `referrer` now behaves as aspected when using back/fwd buttons. 🔥
